packages/playground/src/index.css

```
@@ -1,33 +1,26 @@
-
 @layer base {
   :root {
     --top-bar-height: 40px;
     --color-thumb: var(--surface4);
     --color-thumb-hover: var(--surface6);
     --color-track: transparent;
+    /*
+     * Override playground-ui semantic font tokens with locally-bundled fonts.
+     * Branded Greed (Mastra Cloud) only ships in the platform repo; here we
+     * keep TasaExplorer as the Studio display font to match prior releases.
+     */
```

Stylelint (17.10.0)

[error] Expected empty line before comment (comment-empty-line-before)

(comment-empty-line-before)